pkg/asset/cluster/tfvars.go

```diff
@@ -176,11 +176,25 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		for i, m := range masters {
 			masterConfigs[i] = m.Spec.ProviderSpec.Value.Object.(*azureprovider.AzureMachineProviderSpec)
 		}
+		workers, err := workersAsset.MachineSets()
+		if err != nil {
+			return err
+		}
+		workerConfigs := make([]*azureprovider.AzureMachineProviderSpec, len(masters))
+		for i, w := range workers {
+			// XXX: What does this even mean?
```

Comment:
track, remove this comment before merge.

Reply:
Removed!